client/model.go

```diff
@@ -43,3 +47,38 @@ func (client *Client) Remodel(b []byte) (changeID string, err error) {
 
 	return client.doAsync("POST", "/v2/model", nil, headers, bytes.NewReader(data))
 }
+
+// CurrentModelAssertion returns the current model assertion
+func (client *Client) CurrentModelAssertion() (asserts.Assertion, error) {
+	return currentAssertion(client, "/v2/model")
+}
+
+// CurrentSerialAssertion returns the current serial assertion
+func (client *Client) CurrentSerialAssertion() (asserts.Assertion, error) {
+	return currentAssertion(client, "/v2/model/serial")
+}
+
+// helper function for getting assertions from the daemon via a REST path
+func currentAssertion(client *Client, path string) (asserts.Assertion, error) {
+	q := url.Values{}
+
+	response, err := client.raw("GET", path, q, nil, nil)
+	if err != nil {
+		return nil, fmt.Errorf("failed to query current model assertion: %v", err)
+	}
+	defer response.Body.Close()
+	if response.StatusCode != 200 {
+		return nil, parseError(response)
+	}
+
+	dec := asserts.NewDecoder(response.Body)
```

Comment:
we could also ioutil.ReadAll and then use asserts.Decode. No strong preference.

Reply:
I left it as is, if you want me to change it to use asserts.Decode, let me know